### PR TITLE
Adding combobox role

### PR DESCRIPTION
**Pull request by mnewcomb314**

### CODE REVIEW

File: src/RoleObjectFactory.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import _ from 'lodash';
 import { ROLES } from './Roles';
-import CompositeData from './RoleObjects/CompositeData';
```
Comment (mnewcomb314) [inline src/RoleObjectFactory.js:3]: Just moving this down to line 11 to fit into the order that all the other RoleObjects import use

File: src/RoleObjectFactory.js [hunk 2]
```diff
@@ -135,10 +136,6 @@ function createAccessibilityObjectForRole(config) {
 
       // rest of the roles in alphabetical order by its class entry
 
-    case ROLES.APPLICATION:
```
Comment (mnewcomb314) [inline src/RoleObjectFactory.js:138]: Move to line 159 to actually follow the ordering specified in the comment on line 137

**Review by ryan-fandl — APPROVED** [review]
LGTM

**Review by eprouver — APPROVED** [review]
LGTM

**Review by LinGrin — APPROVED** [review]
LGTM

**Pull request merged**